build.gradle

```diff
@@ -242,7 +243,23 @@ subprojects {
       }
     }
 
-    slf4j = { ->
+
+    slf4jApi = { ->
+      dependencies {
+        compileOnly group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
+      }
+    }
+
+    slf4jLambdaApi = { ->
+      dependencies {
+        slf4jApi()
+        compile(group: 'kr.pe.kwonnam.slf4j-lambda', name: 'slf4j-lambda-core', version: slf4jLambdaApiVersion)  {
```

Comment:
Maybe drop a comment why that's so useful for our case despite being so obscure :)

Reply:
Btw star him guys: https://github.com/kwon37xi/slf4j-lambda

Reply:
@mkondratek FYI, SLF4J 2.0.0 will have this feature as well... but as for now that's alpha only (and even when stable, it's likely gonna be incompatible with f*cking everything existing so far :exploding_head:)